### DISCUSSION TITLE
Cant't change the sync directory

### DISCUSSION THREAD

**Discussion by ykantguillecode** [abraunegg/onedrive#2380] [Q&A]
I recently installed the onedrive client on my Ubuntu 22.04 and i specially made a 200 GB partition in my hard drive to sync my onedrive files. I changed the sync_dir line to the directory of the mounted partition in the config file before compiling and I also checked the ~/.config/onedrive/accounts/(myaccount)/config file after installing and logging in. Despite the sync_dir being establish as the directory of the partition, onedrive still syncs my files to ~/OneDrive. Any idea about how can i fix this?
PD. sorry if i made any grammar mistakes my first language isn't English.

**Comment by abraunegg** [comment]
@guillefdeez 
> I recently installed the onedrive client on my Ubuntu 22.04
> ...
> I changed the sync_dir line to the directory of the mounted partition in the config file before compiling

Hopefully you installed the client correctly following the *correct* installation instructions for Ubuntu as per: https://github.com/abraunegg/onedrive/blob/master/docs/ubuntu-package-install.md

There is zero requirement to self compile for Ubuntu unless you are fixing a bug post the latest release. It is recommended to always use the OpenSuSE Build Service packages to ensure you get the correct client version. 

Do not install the client from the Ubuntu Universe packages. They are obsolete and not supported.

> Despite the sync_dir being establish as the directory of the partition, onedrive still syncs my files to ~/OneDrive. Any idea about how can i fix this?

A couple of questions for you to answer:

1. How are you running the application? Are you using 'sudo'? If you are, do not use 'sudo' with this application.
2. What is the application version you are running? What is the output of `onedrive --version`
3. What is the application configuration? What is the output of `onedrive --display-config`
4. Did you enable any systemd service?
